_pages/en_US/installing-boot9strap-(super-skaterhax).txt

```diff
@@ -42,12 +42,26 @@ In this section, you will copy the files needed to trigger both super-skaterhax
 1. Create a folder named `3ds` on the root of your SD card if it does not already exist
     + This folder stores homebrew applications and data; it is different from the `Nintendo 3DS` folder that the console automatically generates
 1. Copy the `nimdsphax` folder from the nimdsphax `.zip` to the `/3ds/` folder on your SD card
-1. Reinsert your SD card into your console
-1. Power on your console
 
 ![]({{ "/images/screenshots/skater-root-layout.png" | absolute_url }})
 {: .notice--info}
 
+<!-- new section header here? -->
+
+1. Reinsert your SD card into your console
+1. Power on your console
+1. Launch System Settings
+1. Select "Other Settings"
+1. Navigate to "Date & Time" -> "Today's Date"
+1. Tap the Up/Down arrows to set the correct Day, Month and Year
+1. Return to "Other Settings"
```

Comment:
'select ok to confirm' is still necessary here so that people don't back out without saving changes
we might also want to say something for the people that already have the date properly set